src/rust/ensogl/lib/theme/src/lib.rs

```diff
@@ -247,22 +249,22 @@ define_themes! { [light:0, dark:1]
             }
         }
         visualization {
-            background = graph_editor::node::background , Lcha(0.2,0.014,0.18,1.0);
+            background = graph_editor::node::background , graph_editor::node::background;
             text           = Lcha(0.0,0.0,0.0,0.7)   , Lcha(1.0,0.0,0.0,0.7);
             text.selection = Lcha(0.7,0.0,0.125,0.7) , Lcha(0.7,0.0,0.125,0.7);
             error {
                 dataflow.text = Rgba(1.0,0.655,0.141,1.0), Rgba(1.0,0.655,0.141,1.0);
                 panic.text    = Rgba(1.0,0.341,0.125,1.0), Rgba(1.0,0.341,0.125,1.0);
             }
             action_bar {
-                background = Rgba(0.929,0.941,0.953,1.0) , Lcha(0.3,0.014,0.18,1.0); // rgb(237 240 243)
+                background = Rgba(0.929,0.941,0.953,1.0) , Lcha(1.0,0.0,0.0,0.1); // rgb(237 240 243)
```

Comment:
a mysterious comment, here and in many places. Needed some explanation what it is and what it is for. And I'm not sure: if this reflects the value actually set, then it might be that some day the comment will be not synchronized.